aten/src/ATen/native/Resize.h

```diff
@@ -134,6 +134,7 @@ inline void setStrided(
     IntArrayRef size,
     IntArrayRef stride,
     int64_t storage_offset) {
+  TORCH_CHECK(size.size() == stride.size(), "mismatch in length of strides and shape");
```

Comment:
Thanks @gchanan @peterbell10 .

Have addressed the comments.

Also note that have moved the `TORCH_CHECK` at the beginning of `setStrided` as it was seg faulting if it was at the current location of `AT_ASSERT`. (Seg Fault was probably in `checkInBoundsForStorage` but didn't verify.)